messaginginapp/src/test/java/io/customer/messaginginapp/state/MessageBuilderTest.kt

```diff
@@ -0,0 +1,46 @@
+package io.customer.messaginginapp.state
+
+import io.customer.messaginginapp.gist.data.model.GistProperties
+import io.customer.messaginginapp.gist.data.model.Message
+import io.customer.messaginginapp.gist.data.model.MessagePosition
+import io.mockk.every
+import io.mockk.mockk
+
+/**
+ * Creates a configurable, well-mocked Message for testing.
+ * This ensures consistent mock behavior across tests.
+ */
+object MessageBuilderTest {
```

Comment:
Shouldn't it be `MessageBuilderMock` instead as it is not testing but helping mock values?

Reply:
updated